### ISSUE TITLE
TypeScript typings for component

### ISSUE THREAD

**Issue by JKillian** [gpbl/react-day-picker#91]
I made an initial pass at adding some TypeScript typings for your project here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/6825

Feel free to review and suggest changes if you see anything that looks amiss!

**Comment by gpbl** [comment]
Ehy thanks! Nice idea. I'm a bit worried about the additional maintenance 🤔 ... I know I'm not going to update it.

**Comment by gpbl** [comment]
👍 is there something we should add to the README?

**Comment by JKillian** [comment]
Nope, don't think so - TypeScript users will know where to find the typings if they want them
